pypiserver/__main__.py

```diff
@@ -20,7 +20,7 @@
 
 
 def init_logging(level=None, frmt=None, filename=None):
-    logging.basicConfig(level=level, format=frmt)
+    logging.basicConfig(level=level, format=frmt, stream=sys.stdout)
```

Comment:
Great, could you make it also optional and possible to specify from the configuration? Thanks!